src/resource/nonRdfData.ts

```diff
@@ -147,17 +153,15 @@ type SaveFileOptions = GetFileOptions & {
 };
 
 /**
- * Saves a file in a folder at a given URL. The server will return the final
- * filename (which may or may not be the given `slug`), it will return it in
- * the response's Location header.
+ * ```{note} This function is still experimental and subject to change, even in a non-major release.
+ * ```
  *
- * If something went wrong saving the file, the returned Promise will be rejected with an Error.
+ * Saves a file in a folder associated with the given URL. The final filename may or may
+ * not be the given `slug`.
```

Comment:
Do you think it might help to state that we do a POST

Reply:
I wouldn't go into that level of details, since what we want is to add an abstraction layer on top of the standard.